plugins/controlboard/src/ControlBoardDriverOthers.cpp

```diff
@@ -143,6 +126,7 @@ bool GazeboYarpControlBoardDriver::getAmpStatus(int, int *v) //NOT IMPLEMENTED
     return true;
 }
 
+//CONTROL CALIBRATION
 bool GazeboYarpControlBoardDriver::calibrate2(int j, unsigned int iv, double v1, double v2, double v3) //NOT IMPLEMENTED
```

Comment:
I think you need to rebase this PR on top of the latest changes for making yarp3 work. `calibrate2` became `calibrate`. If you never merged branches with possible conflicts, keep a backup of your folder somewhere and try to avoid having non-committed changes.

Reply:
This PR was meant to be done on the master where the changes for YARP3 are not implemented right? (or at least have been reverted https://github.com/robotology/gazebo-yarp-plugins/commit/b7f65bb32b03839f45b3ec69b57e54c869c2e60e).

Do you suggest to work on the dev?

Reply:
I think for the time being we can continue to target `master` .